deepfence_worker/cronjobs/scheduled_tasks.go

```diff
@@ -115,7 +115,7 @@ func runSystemScheduledTasks(ctx context.Context, messagePayload map[string]inte
 		return nil
 	}
 
-	scanTrigger := model.ScanTriggerCommon{NodeIds: nodeIds, Filters: model.ScanFilter{}}
+	scanTrigger := model.ScanTriggerCommon{NodeIds: nodeIds, Filters: model.ScanFilter{}, IsPriority: false}
```

Comment:
Can't we use the request's IsPriority field instead of hard coding `false`?
Better leave the choice to the end user

Reply:
Not sure if this should to be added to the scheduled jobs.

Reply:
This is a cron job, so it should be false.